client/router.js

```diff
@@ -369,3 +427,56 @@ Router.prototype._updateCallbacks = function () {
 
 Router.prototype._page = page;
 Router.prototype._qs = qs;
+
+function shouldCallTrigger(current, fn) {
```

Comment:
I did that because of I don't want to make bigger Router class. How does it affect performance?

Don't we have other way to test :)?